blocks/log.py

```diff
@@ -245,3 +254,18 @@ def __iter__(self):
 
     def get_status(self):
         return self._status
+
+    def to_dataframe(self):
+        return DataFrame.from_dict(self._storage, orient='index')
+
+
+def to_dataframe(log):
```

Comment:
Why not just make this a method of the `Log` class? It acts directly on the class instance, so seems like a perfect candidate.